addons/pos_epson_printer/static/src/js/printers.js

```diff
@@ -24,7 +24,11 @@ var EpsonPrinter = core.Class.extend(PrinterMixin, {
         } else {
             Gui.showPopup('ErrorPopup', {
                 'title': _t('Connection to the printer failed'),
-                'body':  _t('Please check if the printer is still connected, if the configured IP address is correct and if your printer supports the ePOS protocol.'),
+                'body':  _t('Please check if the printer is still connected, if the configured IP address is correct and if your printer supports the ePOS protocol.' +
+                    'Some browsers don\'t allow HTTP calls from websites to devices in the network (for security reasons). ' +
+                    'If it is the case, you will need to follow Odoo\'s documentation for' +
+                    '\'Self-signed certificate for ePOS printers\' and \'Secure connection (HTTPS)\' to solve the issue'
+                ),
```

Comment:
'body': _t('Please check if the printer is still connected, if the configured IP address is correct and if your printer supports the ePOS protocol.\n' +
                        'Some browsers don\'t allow HTTP calls from websites to devices in the network (for security reasons).' +
                        'If it is the case, you will need to follow Odoo\'s documentation for' +
                        '\'Self-signed certificate for ePOS printers\' and \'Secure connection (HTTPS)\' to solve the issue'),

Reply:
![screenshot-10 30 65 75_8069-2022 06 09-14_38_28](https://user-images.githubusercontent.com/38907656/172848891-98e2e267-2f4e-4f4c-9b4d-7dec0b54fe9c.png)

Reply:
@qle-odoo it doesn't work on my side. 
![image](https://user-images.githubusercontent.com/60775325/172849716-9f8e12fb-e1f5-4639-a030-8d52f7cc1bcd.png)
![image](https://user-images.githubusercontent.com/60775325/172849746-ae4ce1c6-5ecf-4e41-be04-19353ebb9582.png)

Reply:
I have suspicion on the fact that it is related to the use of `ErrorPopup`. I do have line breaks using `ErrorTracebackPopup` (probably due to the CSS `white-space: pre-wrap` usage).
`ErrorTracebackPopup` also have the benefit of being able to select the text (for copy paste). Other stuff like "sending" and "downloading" may also be handy.
Do you think it is a good idea to change to this family of popup (or maybe another one ?) ?